### PR TITLE
revise vasp.py 

### PR DESCRIPTION
**Pull request by apalizha**
1. users now have the option to specify points
2. included the vasp flags needed for bulk optimizations originally used for oc20 bulks.

### CODE REVIEW

File: ocdata/vasp.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 from ase.calculators.vasp import Vasp2
 from ase.calculators.singlepoint import SinglePointCalculator as SPC
 
-
+# NOTE: this is the setting for slab and adslab
 VASP_FLAGS = {'ibrion': 2,
               'nsw': 200,
```
Comment (janiceblue) [inline ocdata/vasp.py:20]: While we're modifying this file, should we change the default to NSW = 2000 as well?

File: ocdata/vasp.py [hunk 2]
```diff
@@ -31,6 +31,20 @@
               'pp': 'PBE',
               'xc': 'PBE'}
 
+# This is the setting for bulk optmization.
+# Only use when expanding the bulk_db with other crystal structures.
+BULK_VASP_FLAGS = {'ibrion': 1,
```
Comment (janiceblue) [inline ocdata/vasp.py:36]: This is currently unused in the code - did you want to just leave this here as a reference, or do we want to add a command line arg for users to optionally replace the lines of `vasp_flags = VASP_FLAGS.copy()` with `vasp_flags = BULK_VASP_FLAGS.copy()`? The latter would be cleaner, but I'm not sure if we'll be using this often enough to warrant changing command line args.

File: ocdata/vasp.py (hunk 1 above)
Comment (apalizha) [reply ocdata/vasp.py:20]: Sure, that's a good idea. I will update that

File: ocdata/vasp.py (hunk 2 above)
Comment (apalizha) [reply ocdata/vasp.py:36]: Mostly want to leave it there as a reference. The use case would be input that as an arg to line 35 when we need it for bulk optimizations.  I agree it's not super clean as it is. I can move it to .CONSTANT folders if you think that's better?

Comment (janiceblue) [reply ocdata/vasp.py:36]: I'm OK leaving this here (or move both dicts to constants.py) as a reference for now. If we find ourselves needing to run with different flags often in the future, then we can move this to a command line arg.

Comment (apalizha) [reply ocdata/vasp.py:36]: @janiceblue updated nsw, left bulk_vasp_flags in vasp.py for now until something changes. Ready to merge if everything else looks good to you.

Comment (janiceblue) [reply ocdata/vasp.py:36]: Sounds good, LGTM

**Pull request merged**